### PR TITLE
Update success page

### PR DESCRIPTION
**Pull request by luigi**
application submitted 2018-08-16 16-09-53

application submitted 2018-08-16 16-10-16

### CODE REVIEW

File: app/views/integrated/application_submitted/edit.html.erb [hunk 1]
```diff
@@ -76,8 +74,21 @@
         </div>
       </div>
     </div>
+    <%= form_for @feedback_form, url: feedback_path, builder: MbFormBuilder, remote: true do |f| %>
```
Comment (luigi) [inline app/views/integrated/application_submitted/edit.html.erb]: Using `form_with` didn't seem to AJAX-ify the form, but passing `remote: true` to `form_for` worked correctly.

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -237,6 +237,21 @@ var noneOfTheAbove = (function() {
   }
 })();
 
+var feedbackForm = (function() {
+  var feedback = {
+    init: function () {
+      $('body').on('ajax:success', function(event, xhr, status, error) {
+        var success = '<span class="radio-button__image emoji emoji--med emoji--blush">Thank you</span>' +
+          '<h3>Thanks for your feedback! It will help improve this application for people in the future.</h3>';
+        $('#feedback-form').html(success);
```
Comment (luigi) [inline app/assets/javascripts/application.js]: Fully replacing the HTML with JS.

File: spec/features/integrated_application/integrated_application_with_single_member_spec.rb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 require "rails_helper"
 
 RSpec.feature "Integrated application" do
-  scenario "with one member", :a11y do
+  scenario "with one member", :js do
```
Comment (luigi) [inline spec/features/integrated_application/integrated_application_with_single_member_spec.rb]: Remove `:a11y` means this feature spec doesn't check for accessibility. I don't think that's ideal, but it's more ideal than making a duplicate spec that does.

Comment (hartsick) [reply spec/features/integrated_application/integrated_application_with_single_member_spec.rb]: yeah not ideal, but I think better than the alternative

**Comment by hartsick** [comment]
Unfortunately, I didn't end up finishing this—ran into a couple of snags:

I found out that the current way we're doing this doesn't allow a user to keyboard navigate to the fields. I think the culprit is the input[type="radio"] { display: none }. This came up when I was trying to make accessible error messages, which is a bit painful because we've never done this with Javascript before and our form builder methods aren't set up for doing this easily.

I looked at GCF's code for their feedback survey, and saw a few differences. They:
1. Don't validate a rating (so no errors, period).
1. Don't submit with JS, and instead re-render the page with a different message.
1. Can't access the emoji radio buttons via tab (not accessible).

I talked with Andrew about the last bullet, and it sounds like none of their radio buttons are accessible by tabbing; it's on their todos and they're interested in if we were able to come up with a pattern they could use.

So, for this story, I think we have a couple of options:
1. Move forward knowing it's inaccessible (with or without errors).
1. Use our JS approach, but work to make it accessible (change the way we render buttons and errors, if we move forward with errors)
1. Switch to re-rendering the HTML (maybe by having two forms). We could use our standard form builder to render errors if we were to move forward.

I'm personally pro #2 with removing any validation that we have on the feedback form object.

I'd love to hear your thoughts!

**Comment by hartsick** [comment]
I'm going to close this PR, since it's been handled in https://github.com/codeforamerica/michigan-benefits/pull/868 & https://github.com/codeforamerica/michigan-benefits/pull/869

**Pull request closed without merging**